src/components/DetailMap/DetailMap.vue

```diff
@@ -193,7 +196,7 @@
         if (!reservoir) {
           return
         }
-        const { fid } = reservoir.properties
+        const fid = reservoir.properties?.fid || reservoir.id
```

Comment:
The `/reservoir/geometry` endpoint doesn't include the `fid` on the reservoir properties, so I don't have that value. Do you think I should ask Jaap to change it? 

I also check and we could be using the `reservoir.id` on all cases, but I wasn't sure why we didn't already. Do you happen to know?